intro-HPC/ch_VNC.tex

```diff
@@ -0,0 +1,132 @@
+\chapter{Graphical applications with VNC}
+\label{ch:vnc}
+
+It's important to remember that VNC sessions are permanent. They survive network
+problems and (unintended) connection loss. This means you can logout and go home
+without a problem (like the terminal equivalent \lstinline|screen| or \lstinline|tmux|).
+This also means you don't have to start \lstinline|vncserver| each time you want to connect.
+
+\section{Start the VNC server}
+\label{sec:start-vnc}
+
+First login on the login node (see \autoref{sec:first-time-connection-to-the-hpc})
+or to the worker node where your job has started. % TODO: where is this described?
```

Comment:
If it's not described yet, write a section on it.

`qstat -an` shows hostname of workernode

For UGent, you'll need to add `<clustername>.os`, so `node2001` becomes `node2001.delcatty.os` to log in

Reply:
I could have sworn I've read about SSH'ing into a workernode, but I can't find it anymore. Guess I'll just add a section on it. Where should I do it? Maybe in `ch_running_interactive_jobs.tex`?

Reply:
It's relevant for both, also with a regular job people should know how to log in to the workernode where their job is running (so they can check what it's doing).

With interactive jobs, no `ssh` is needed, you get a shell on a workernode...

Reply:
hmmm, maybe we should fix this so the domainname is not needed

Reply:
Fix this in the documentation (with a site-specific macro) or over at your end? @stdweird

Reply:
@redfast00 He means on the system, so people can use domain-less hostname like `ssh node2001`

Reply:
on our end. shouldn't be that hard.

Reply:
you can now ssh to short hostnames from gent loginnodes